### PR TITLE
Update ApplicationResources.properties

### PR DESCRIPTION
**Pull request by 1somya**

### CODE REVIEW

**Comment by 1somya** [comment]
Changes made according to the trends seen in the file.

**Comment by 1somya** [comment]
And the necessary things

File: xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties [hunk 1]
```diff
@@ -199,8 +199,8 @@ allow_0=Deny
 allow_1=Allow
 from=From
 to=To
-editedby=edited by
```
Comment (tmortagne) [inline xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties]: I'm pretty sure the lower case is intended here.

File: xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties [hunk 2]
```diff
@@ -199,8 +199,8 @@ allow_0=Deny
...
-on=on
```
Comment (tmortagne) [inline xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties]: Same here.

File: xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties [hunk 3]
```diff
@@ -252,20 +252,20 @@ forcelock=Click here to force editing
 initialversion=Initial Version
 rollback=Rollback
 readytorollback=Do you want to rollback to version
-readonly=This server is currently in read-only mode
+readonly=This server is currently in read-only mode.
```
Comment (tmortagne) [inline xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties]: Not sure this is correct, would need to check how it looks in the UI.

File: xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties [hunk 4]
```diff
@@ -252,20 +252,20 @@ forcelock=Click here to force editing
...
 revisiondoesnotexist=This page does not exist in this version.
 nocommentwithnewdoc=You cannot comment on a page or article that does not exist.
 actiondoesnotexist=This action does not exist!
 thiswikidoesnotexist=This Wiki does not exist on this server.
 thispagedoesnotexist=The requested page could not be found.
-nosuchobject=The specified object does not exist
+nosuchobject=The specified object does not exist.
```
Comment (tmortagne) [inline xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties]: Not sure this is correct, would need to check how it looks in the UI.

File: xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties [hunk 5]
```diff
@@ -252,20 +252,20 @@ forcelock=Click here to force editing
...
 thispagealreadyexists=This page already exists.
 attachmentdoesnotexist=The attachment does not exist.
 wikicontentcannotbeempty= The content of a wiki page is not allowed to be completely empty.
 fileuploadislarge=XWiki has a default limit of around 10Mb for attached files. This limit can be changed using the upload_maxsize parameter. Check the FAQ for more information.
 javaheapspace=Java Heap Space Out Of Memory Exception!
 notsupportcharacters=File name does not support characters '\\' '/' ';'
-thistemplatedoesnotexist=This template does not exist
+thistemplatedoesnotexist=This template does not exist.
```
Comment (tmortagne) [inline xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties]: Not sure this is correct, would need to check how it looks in the UI.

**Comment by 1somya** [comment]
What about the 6th one? 
Actually I have made 6 changes

**Comment by tmortagne** [comment]
No comment is a good sign :)

I don't really have anything specific to say about last change, I only comment things for which I think there is a mistake or something to improve.

**Comment by 1somya** [comment]
okay so that means 1 is done?
What about the other one which you are not sure about?Should I consider them or not?

**Comment by tmortagne** [comment]
Thanks for your contribution.

**Pull request merged**